src/components/layouts/Shared/HeaderNav.tsx

```diff
@@ -108,15 +118,40 @@ const headerNavItemStyles = css`
     display: inline-block;
     width: auto;
     margin-left: 2rem;
+  }
+`;
 
-    &:first-of-type {
-      margin-left: 0;
-    }
+const HeaderNavLinkSubitem: AnyStyledComponent = styled(Link)`
+  position: absolute;
+  right: 0;
+  display: block;
+  font-family: inherit;
+  color: #ffffff;
+  text-decoration: none;
+  font-size: 0.9rem;
+  font-weight: 600;
+  line-height: 3rem;
+
+  @media (min-width: 600px) {
+    display: inline-block;
+    width: auto;
   }
 `;
 
-const HeaderNavLinkItem: AnyStyledComponent = styled(Link)`
-  ${headerNavItemStyles}
+const HeaderNavItemWithSubitems: AnyStyledComponent = styled.div`
```

Comment:
please check alignment. they should be aligned with the parent

Reply:
left aligned? because they are right-aligned right now

Reply:
![Apr-16-2020 16-05-05](https://user-images.githubusercontent.com/339804/79465865-29eb6400-7ffc-11ea-9be6-48d77912aa30.gif)

yeah let's do left for now